node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,16 @@
+import fs from 'node:fs/promises';
+
+const file = process.argv[2];
+// console.log('argv: ', process.argv);
+// console.log('file:', file);
+
+async function readAny() {
+  const contents = await fs.readFile(`./${file}`, { encoding: 'utf8' });
```

Comment:
The `./` isn't necessary and makes it so you can't use absolute paths. It's best to let the user type `./` into the command line if it's necessary.